src/Symfony/Component/DependencyInjection/Definition.php

```diff
@@ -661,9 +661,15 @@ public function isPublic()
      * @return Definition The current instance
      *
      * @api
+     *
+     * @deprecated since version 2.7, will be removed in 3.0.
      */
-    public function setSynchronized($boolean)
+    public function setSynchronized($boolean, $triggerDeprecationError = true)
```

Comment:
It's not on interface so if you overwrite class you don't need to copy params, yet this argument is optional, so I guess it's not real BC break.